thoth/package_extract/image.py

```diff
@@ -477,17 +496,15 @@ def _get_python_interpreters(path: str) -> dict:
             if len(parts) == 2 and parts[0] == "Python":
                 version_ = line.rstrip()
         except Exception as exc:
-            _LOGGER.warning(
-                "Failed to get python version: %s", str(exc))
+            pass
+
+        absolute_link = _get_absolute_link(path, py_path, 0)
 
         py_interpret = {
             "path": py_path[len(path):],
-            "link": None,
+            "link": absolute_link[len(path):] if absolute_link != py_path else None,
```

Comment:
This takes care of removing temporary folder path at the beginning and if the symlink does not point to another symlink, it would just assign `None` value to it.